src/opentelemetry/launcher/configuration.py

```diff
@@ -223,10 +283,8 @@ def configure_opentelemetry(
             == _DEFAULT_OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
```

Comment:
there should probably be a check in here for the combination of LS_ACCESS_TOKEN and the metrics endpoint

Reply:
Right, fixed